apps/central-scan/frontend/src/app_root.tsx

```diff
@@ -460,10 +473,41 @@ export function AppRoot({
     }
   }, [electionJustLoaded, usbDrive.status]);
 
+  const authStatus = authStatusQuery.data;
+
+  useEffect(() => {
+    async function configure() {
+      if (
+        !configureMutation.isLoading &&
+        !electionDefinition &&
+        authStatus &&
+        isElectionManagerAuth(authStatus) &&
+        usbDrive.status === 'mounted'
+      ) {
+        const result = await configureMutation.mutateAsync();
+        const electionDefinitionFromResult = result.ok();
+        assert(electionDefinitionFromResult !== undefined);
+        updateElectionDefinition(electionDefinitionFromResult);
+      }
+    }
+
+    void configure();
+  }, [
+    configureMutation,
+    authStatus,
+    electionDefinition,
+    updateElectionDefinition,
+    usbDrive,
+  ]);
+
+  // The `configure` useEffect hook above depends on `authStatus`. Since hooks can't
+  // be rendered conditionally, we need to short circuit on authStatusQuery after the
+  // hook is defined.
   if (!authStatusQuery.isSuccess) {
     return null;
   }
-  const authStatus = authStatusQuery.data;
+  // assert authStatus so TypeScript knows it's not undefined
+  assert(authStatus);
```

Comment:
Super minor but I think you can drop this assertion if you move the defining of `authStatus` back here:

```ts
const authStatus = authStatusQuery.data;
```

And just use `authStatusQuery.data` in the `useEffect` above, which might be preferable since the `useEffect` will eventually go away when we fully move the election definition to react-query.

But also feel free to leave as is!

Reply:
Good call - [5057e2d](https://github.com/votingworks/vxsuite/pull/3304/commits/5057e2dfb8907c5edb394aad4c0b2ff0373e332c)